### ISSUE TITLE
Loot in Scenario Summary only counts coins

### ISSUE THREAD

**Issue by tttopuz** [Lurkars/gloomhavensecretariat#455]
### Describe the bug

At the end of a scenario, in the scenario summary window, the row Loot only counts coins, whereas it should count the number of loot tokens collected.
![image](https://github.com/Lurkars/gloomhavensecretariat/assets/6255709/65a2881c-b59b-45d3-8e0b-dbf3f5bb9309)

### Game Enviroment

-

### To Reproduce

-

### Expected behavior

-

### Gloomhaven Secretariat Version

-

### Used Browser

-

### OS

Windows

**Comment by Jaerin** [comment]
I'm not sure there is a time when you'd need a conversation of items to gold value at least in this situation.  The question I have though is in a smaller group where a loot card counts as more than one material does that count as multiple loot cards with regards to personal quests and the like?  I could see this being a reason why you'd need to calculate the "value" of the loot card, but I'm not sure on the ruling on that.  That was the only reason I could come up with why you'd need to calculate the value of anything other than coins.

**Comment by tttopuz** [comment]
In my opinion, the number of loot tokens collected should be displayed here. There are for example some battle goals where you need to compare the number of collected loot tokens between players. Actual coins collected is much less important in Frosthaven.

**Comment by Lurkars** [comment]
Yes, I also think that showing just the total number of collected tokens (e.g. collected loot cards) is here a way better information. In Gloomhaven it's also the number of loot tokens, so showing the number of coins here is just confusing. So this will change in next release. For now I will just skip the info of collected coins. If this is ever required somehow, I can rethink about where to show it.

**Comment by Lurkars** [comment]
Add in [v0.83.4](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.83.4). Happy about feedback and if issue is solved with this. Thank.

**Comment by tttopuz** [comment]
Top row now displays number of loot tokens. Thanks!
